src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -26,11 +32,68 @@ public IView? View
 
 		bool HasFixedConstraints => CrossPlatformLayout is IConstrainedView { HasFixedConstraints: true };
 
+		bool ShouldIgnoreSafeAreaForEdge(int edge)
```

Comment:
You need to implement the new flags that are inside `SafeAreaRegions`

You will most likely need to change the implementation of ISafeAreaPage.IgnoreSafeAreaForEdge

It will probably just need to be changed so it returns the SafeAreaEdges that have been specified and then you can inspect what was returned in order to determine how much to inset the view by